RFS/src/main/java/org/opensearch/migrations/bulkload/transformers/Transformer_ES_6_8_to_OS_2_11.java

```diff
@@ -64,6 +64,7 @@ public ObjectNode getRawJson() {
                         .addArgument(indexTemplate::getRawJson)
                         .setCause(e)
                         .log();
+                    throw e;
```

Comment:
When using the metadata transformation process we actually check if no value is provided and error out with `No multi type resolution behavior declared, specify --multi-type-behavior to process` however! this error was being swallowed and not making it up to the caller or failing the migration.  Making sure this kind of failure is prominently featured.

BTW is exactly the behavior I'd recommend we have with the typeMappingsSanitizer which has been part of discussions in https://opensearch.atlassian.net/browse/MIGRATIONS-2416

Reply:
Oof, good catch.